app/parsers/bulkrax/application_parser.rb

```diff
@@ -134,7 +134,9 @@ def write_import_file(file)
 
     # Base path for imported and exported files
     def base_path(type = 'import')
-      ENV['HYKU_MULTITENANT'] ? File.join(Bulkrax.send("#{type}_path"), Site.instance.account.name) : Bulkrax.send("#{type}_path")
+      # account for multiple versions of hyku, but don't change the import paths
+      is_multitenant = ENV['HYKU_MULTITENANT'] == 'true' || ENV['SETTINGS__MULTITENANCY__ENABLED'] == 'true'
```

Comment:
pals is using `ENV['SETTINGS__MULTITENANCY__ENABLED']`

louisville, british libraries and utk are all using `ENV['HYKU_MULTITENANT']`. I believe the difference is the hyku version. so I'm accounting for both so multiple versions of hyku still work as expected.

Reply:
What would happen if an application started as single tenant, then later decided to go multitenant?  Is that ever an edge to worry about?

Reply:
the data would still exist in the db, they just wouldn't be able to access the path anymore.

for importing -- this shouldn't really be a problem. once the data is imported...I don't see a great need to access the imported file again.

for exporting -- re run the exporter, or create a new one.